src/form/input/file/FileInput.tsx

```diff
@@ -18,57 +18,59 @@ export interface FileInputProps {
   customClassName?: string;
   customLabelClassName?: string;
   acceptedFileTypes?: string;
-  labelRef?: React.RefObject<HTMLLabelElement>;
+  ref?: React.RefObject<HTMLDivElement>;
 }
 
-function FileInput({
-  onChange,
-  children,
-  testid,
-  name,
-  htmlFor,
-  acceptedFileTypes = "image/png, image/jpeg, .pdf",
-  customSpinner,
-  customClassName,
-  customLabelClassName,
-  isPending,
-  isDisabled,
-  labelRef
-}: FileInputProps) {
-  const containerClassName = classNames("file-input__container", customClassName);
-  const isInputDisabled = isPending || isDisabled;
-  const labelClassName = classNames("file-input__label", customLabelClassName, {
-    "file-input__label--is-disabled": isInputDisabled
-  });
-  const spinnerContent = customSpinner || (
-    <Spinner customClassName={"file-input__spinner"} />
-  );
+const FileInput = React.forwardRef<HTMLDivElement, Record<string, any>>(
```

Comment:
I think it makes more sense for this component to forward this ref to label tag and remove labelRef prop

Reply:
Because we would probably need to access the label instead of the container